Update citation and add BEAM section to README (#8)
PlasmaFAIR/epydeck: README.md

# epydeck

![PyPI](https://img.shields.io/pypi/v/epydeck?color=blue)
![Build/Publish](https://github.com/PlasmaFAIR/epydeck/actions/workflows/build_publish.yml/badge.svg)
![Tests](https://github.com/PlasmaFAIR/epydeck/actions/workflows/tests.yml/badge.svg)

> An open-source Python project which acts as an [EPOCH](https://epochpic.github.io/) input file (deck) reader/writer. A module of [BEAM](#broad-epoch-analysis-modules-beam) (Broad EPOCH Analysis Modules).

Reviewer: A little clearer:

```suggestion
Epydeck is an [EPOCH](https://epochpic.github.io/) input file (deck) reader/writer. Part of [BEAM](#broad-epoch-analysis-modules-beam) (Broad EPOCH Analysis Modules).
```

  }
}
```

</details>

## Citing

If epydeck contributes to a project that leads to publication, please acknowledge this by citing epydeck. This can be done by clicking the "cite this repository" button located near the top right of this page.

## Broad EPOCH Analysis Modules (BEAM)

> ![[BEAM.png]]

Reviewer: This is the wrong syntax for github markdown, I think it should be:

```suggestion
![BEAM logo](./BEAM.png)
```
